src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float LOAD_FACTOR = 0.75f;
+
+    private int capasity;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        capasity = DEFAULT_CAPACITY;
+        size = 0;
+        table = new Node[capasity];
+    }
+
+    static class Node<K,V> {
```

Comment:
From the checklist: 'Remember to parameterize the Node class'. So, please, parameterize the Node class with <K,V>.